server.go

```diff
@@ -24,5 +24,5 @@ func main() {
 
 	n := negroni.Classic()
 	n.UseHandler(router)
-	n.Run(":3000")
+	n.Run(":3001")
```

Comment:
Oops, my bad. I didn't realize that. It can be changed to whatever, but it
may be a good idea to have the frontend be 4000 like you mentioned.
Completely up to you, and I'm happy to change this if needed.

On Wed, May 27, 2015, 6:44 PM Francois Mariette <notifications@github.com>
wrote:

> In server.go
> <https://github.com/EaseApp/web-backend/pull/7#discussion_r31197390>:
>
> > @@ -24,5 +24,5 @@ func main() {
> >
> >  	n := negroni.Classic()
> >  	n.UseHandler(router)
> > -	n.Run(":3000")
> > +	n.Run(":3001")
>
> Is this going to stay at 3001? Currently the front end is using 3001 but I
> can change it to whatever as well. Just wanted to know beforehand. Maybe we
> should have the front end use port 4000 so that we can increment easily.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/EaseApp/web-backend/pull/7/files#r31197390>.
>